Dockerfile

```diff
@@ -11,27 +11,32 @@ ARG USER_ID=1000
 # but rather use the `user` option in docker-compose.yml instead
 USER root
 
-# Allow scripts to detect we're running in our own container
-RUN touch /addons-server-docker-container
-
-# Add nodesource repository and requirements
-ADD docker/nodesource.gpg.key /etc/pki/gpg/GPG-KEY-nodesource
+# Add support for https apt repos and gpg signed repos
 RUN apt-get update && apt-get install -y \
         apt-transport-https              \
         gnupg2                           \
     && rm -rf /var/lib/apt/lists/*
-RUN cat /etc/pki/gpg/GPG-KEY-nodesource | apt-key add -
-ADD docker/debian-buster-nodesource-repo /etc/apt/sources.list.d/nodesource.list
-ADD docker/debian-buster-backports-repo /etc/apt/sources.list.d/buster-backports.list
+# Add keys and repos for node and mysql
+COPY docker/*.gpg.key /etc/pki/gpg/
+RUN APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=DontWarn \
```

Comment:
This is new, right?

Reply:
This is yak shaving : I'm adding a new gpg key, but I didn't want to do that using 2 docker commands, so I passed it using a file instead of stdin, but I was annoyed at the output of `apt-key`, so I use that to shut it down.